### PR TITLE
Fix incorrect printf format specifier and typo

### LINKED ISSUE #5532: [tools] radius_filter tool segfaults when run without arguments
https://github.com/PointCloudLibrary/pcl/blob/58f9cfb41115b1f2453dfe0ec75915243969fb66/tools/radius_filter.cpp#L60

This is on macos 12.5.1 with a current master build. When run without arguments, this crashes here, although its not clear to me why.

```
(lldb) r
Process 48825 launched: 'foobar/deps/bin/pcl_radius_filter' (arm64)
Filter a point cloud using the pcl::TfQuadraticXYZComparison. For more information, use: foobar/deps/bin/pcl_radius_filter -h
Syntax is: foobar/deps/bin/pcl_radius_filter input.pcd output.pcd <options>
  where options are:
Process 48825 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x3ff0000000000000)
    frame #0: 0x0000000198fe0864 libsystem_platform.dylib`_platform_strlen + 4
libsystem_platform.dylib`:
->  0x198fe0864 <+4>:  ldr    q0, [x1]
    0x198fe0868 <+8>:  adr    x3, #-0xc8                ; ___lldb_unnamed_symbol314
    0x198fe086c <+12>: ldr    q2, [x3], #0x10
    0x198fe0870 <+16>: and    x2, x0, #0xf
Target 0: (pcl_radius_filter) stopped.
(lldb) bt
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x3ff0000000000000)
  * frame #0: 0x0000000198fe0864 libsystem_platform.dylib`_platform_strlen + 4
    frame #1: 0x0000000198e8f6a0 libsystem_c.dylib`__vfprintf + 4544
    frame #2: 0x0000000198ec1ac8 libsystem_c.dylib`vfprintf + 92
    frame #3: 0x00000001017c74b8 libpcl_common.1.12.dylib`pcl::console::print_value(char const*, ...) + 132
    frame #4: 0x000000010000b8b8 pcl_radius_filter`printHelp(int, char**) + 88
    frame #5: 0x00000001000118f0 pcl_radius_filter`main + 76
    frame #6: 0x000000010002908c dyld`start + 520
```

### PR DESCRIPTION
**Pull request by themightyoarfish**
fixes #5532

### CODE REVIEW

File: tools/radius_filter.cpp [hunk 1]
```diff
@@ -57,8 +57,8 @@ printHelp (int, char **argv)
 {
   pcl::console::print_error ("Syntax is: %s input.pcd output.pcd <options>\n", argv[0]);
   pcl::console::print_info ("  where options are:\n");
-  pcl::console::print_info ("                     -radius X = Radius of the spere to filter (default: ");
-  pcl::console::print_value ("%s", default_radius); pcl::console::print_info (")\n");
+  pcl::console::print_info ("                     -radius X = Radius of the sphere to filter (default: ");
+  pcl::console::print_value ("%d", default_radius); pcl::console::print_info (")\n");
```
Comment (mvieth) [inline tools/radius_filter.cpp:61]: `default_radius` is a float, so `%d` is also wrong (`%d` is for signed decimal integers). For float and double, `%f` or `%g` would be appropriate. https://cplusplus.com/reference/cstdio/printf/

**Review by larshg — APPROVED** [review]

File: tools/radius_filter.cpp (hunk 1 above)
Comment (larshg) [inline tools/radius_filter.cpp:61]:
```suggestion
  pcl::console::print_value ("%f", default_radius); pcl::console::print_info (")\n");
```

Comment (larshg) [reply tools/radius_filter.cpp:61]: As @mvieth said 😄 just did this for easier commit using the web ui.

Comment (themightyoarfish) [reply tools/radius_filter.cpp:61]: god dammit

**Review by larshg — APPROVED** [review]

**Review by mvieth — APPROVED** [review]
Thank you!

**Pull request merged**